drivers/storage/portworx/component/securitycontextconstraints.go

```diff
@@ -0,0 +1,147 @@
+package component
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/go-version"
+	ocp_secv1 "github.com/openshift/api/security/v1"
+	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/record"
+	"k8s.io/utils/pointer"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	pxutil "github.com/libopenstorage/operator/drivers/storage/portworx/util"
+	opcorev1 "github.com/libopenstorage/operator/pkg/apis/core/v1"
+)
+
+const (
+	// SCCComponentName name a component for installing securityContextConstraints.
+	SCCComponentName = "scc"
+	// PxSCCName name of portworx securityContextConstraints
+	PxSCCName = "portworx"
+)
+
+type scc struct {
+	k8sClient client.Client
+}
+
+func (s *scc) Name() string {
+	return SCCComponentName
+}
+
+func (s *scc) Priority() int32 {
+	return int32(0)
+}
+
+func (s *scc) Initialize(k8sClient client.Client, k8sVersion version.Version, scheme *runtime.Scheme, recorder record.EventRecorder) {
+	s.k8sClient = k8sClient
+}
+
+func (s *scc) IsPausedForMigration(cluster *opcorev1.StorageCluster) bool {
+	return false
+}
+
+func (s *scc) IsEnabled(cluster *opcorev1.StorageCluster) bool {
+	ocp_secv1.Install(s.k8sClient.Scheme())
```

Comment:
Q: What does this Install do?

Reply:
client-go does not recognize the CRD type without it.

Reply:
install is a replacement of AddToScheme, AddToScheme is deprecated by ocp

Reply:
Oh got it. It is adding the SecurityContextConstraint object to scheme. Makes sense.
The next line is adding the CRD type.